docs/ossm/ossm2-migration/network-policies.md

```diff
@@ -0,0 +1,144 @@
+# OpenShift Service Mesh 2.6 migration to 3.0
+
+In OpenShift Service Mesh 2.6, Network Policies are created by default when `spec.security.manageNetworkPolicy=true` in the ServiceMeshControlPlane config. During migration to Service Mesh 3.0, these Network Policies will be removed and will need to be recreated manually if you wish to maintain identical NetworkPolicies.
+
+## Network Policies created by 2.6:
+
+When `spec.security.manageNetworkPolicy=true`, the following Network Policies are created:
+
+### 1. Istiod Network Policy
+- **Purpose**: Controls incoming traffic to the webhook port of istiod pod(s)
+- **Location**: Created in SMCP namespace
+- **Sample YAML**:
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: istio-istiod-default    # Name format: istio-istiod-<revision>
+  namespace: istio-system       # Your SMCP namespace
+  labels:
+    maistra-version: "2.6.5"    # Version label
+    app: istiod                 # Identifies istiod component
+    istio: pilot                # Identifies as Istio pilot component
+    istio.io/rev: default       # Revision identifier
+    release: istio              
+  annotations:
+    "maistra.io/internal": "true"
+spec:
+  podSelector:
+    matchLabels:
+      app: istiod
+      istio.io/rev: default
+  ingress:
```

Comment:
Is this from multitenatnt 2.6? In cluster wide, I see this:
```
spec:
  podSelector:
    matchLabels:
      app: istiod
      istio.io/rev: basic
  ingress:
    - {}
  policyTypes:
    - Ingress
```

Reply:
Good catch, verified and added https://github.com/openshift-service-mesh/sail-operator/pull/203/commits/19e47dd1d7ea6a8157f1a8d2b92a54909692a205